backend/src/controllers/ProjectController.ts

```diff
@@ -52,6 +52,16 @@ class ProjectController {
     return project;
   }
 
+  async lookup(pageSize: number): Promise<ProjectDoc[]> {
+    const projects = await this.projectModel
+      .find()
```

Comment:
Looks good overall :) however, without an additional criteria in the `find()` clause, this will only ever return the oldest projects created. Alternative is to sort the ID in descending order so we get the latest projects created in the preview

Reply:
Ok, makes sense, thanks. I'll take a look either tonight after work or tomorrow

Reply:
Added `.sort({ _id: -1 })` it appears to work